doc/telemetry_config.md

```diff
@@ -53,7 +53,7 @@ You may use all, some, or none of the collectors. The following collectors suppo
 #### `In-Mem`
 | Configuration | Type   | Description           | Default |
 |---------------|--------|-----------------------|---------|
```

Comment:
may we keep following the table format that was introduced recently?

Reply:
Oh yep. Thanks for pointing that out. Fixed.